index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+      type="text/css"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <a href="#" class="logo">Abimbola</a>
+        <a><i class="fa fa-bars cursor"></i></a>
+      </nav>
+    </header>
+
+    <section class="main-section img">
+        <div>
+          <h1 class="greetings">Hey There.
+            <span class="greetings">I'm Abimbola</span>
```

Comment:
- [ ] The subtitle should use a header tag hx, where x:- 2, 3, 4, or 5. 

Always try to use hx if you have to use it for title or subtitle as this helps with your page's readability and optimization.